products/veeam-backup-and-replication

```diff
@@ -0,0 +1,123 @@
+---
+title: Veeam Backup & Replication
+category: app
+tags: veeam
+iconSlug: veeam
+permalink: /veeam-backup-and-replication
+alternate_urls:
+-   /veeam-backup
+releasePolicyLink: https://www.veeam.com/product-lifecycle.html
+activeSupportColumn: End of Fix
+releaseColumn: false
+releaseDateColumn: true
+eolColumn: End of support
+
+releases:
+-   releaseCycle: "12.0"
+    releaseDate: 2023-01-30
+    support: true
+    eol: 2026-02-01
+    link: "https://www.veeam.com/kb4420"
+
+-   releaseCycle: "11.0a"
+    releaseDate: 2021-10-07
+    support: 2023-02-01
+    eol: 2024-02-01
+    link: "https://www.veeam.com/kb4245"
+
+-   releaseCycle: "11.0"
+    releaseDate: 2021-02-11
+    support: 2023-02-01
+    eol: 2024-02-01
+    link: "https://www.veeam.com/kb4126"
+
+-   releaseCycle: "10.0"
+    releaseDate: 2020-02-04
+    support: 2021-02-01
+    eol: 2023-02-01
+    link: "https://www.veeam.com/kb2680"
+
+-   releaseCycle: "9.5"
+    releaseDate: 2016-11-16
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
+
+-   releaseCycle: "9.0"
+    releaseDate: 2016-01-12
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
+
+-   releaseCycle: "8.0"
+    releaseDate: 2014-11-06
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
+
+-   releaseCycle: "7.0"
+    releaseDate: 2013-08-20
+    support: false    
+    eol: true
+    link: "https://www.veeam.com/kb2680"
+
+-   releaseCycle: "6.5"
+    releaseDate: 2012-10-09
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
+
+-   releaseCycle: "6.1"
+    releaseDate: 2012-06-04
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
+
+-   releaseCycle: "6.0"
+    releaseDate: 2011-08-22
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
+
+-   releaseCycle: "5.0"
+    releaseDate: 2010-08-30
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
+
+-   releaseCycle: "4.0"
+    releaseDate: 2009-10-29
```

Comment:
Where can I confirm this date?

Reply:
[This](https://en.wikipedia.org/wiki/Veeam_Backup_%26_Replication) has some information.